maquetado/signup.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Sign Up</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+    <form class="info-container column">
+      <img src ="./assets/wolox.png" class="login-img" alt="Wolox">
+      <label for="name" type="submit"class="input-label">Nombre</label>
+      <input name="name" id="name" class="input-field" type="text">
+      <label for="surname" type="submit"class="input-label">Apellido</label>
+      <input name="surname" id="surname" class="input-field" type="text">
+      <label for="email" type="submit"class="input-label">Email</label>
+      <input name="email" id="email" class="input-field" type="email">
+      <label for="password" type="submit"class="input-label">Password</label>
+      <input name="password" id="password" class="input-field" type="password">
+      <label for="password-confirmation" type="submit"class="input-label">Confirmación de Password</label>
```

Comment:
Don't add type to labels

```suggestion
      <label for="password-confirmation" class="input-label">Confirmación de Password</label>
```